docs/changelog.md

```diff
@@ -152,6 +152,10 @@ If you however want to continue using the `DefaultGraphManager`, then you can si
 setting again or change it to the TinkerPop `GraphManager` that has been the default before:
 `org.apache.tinkerpop.gremlin.server.util.DefaultGraphManager`.
 
+##### Remove support for old serialization format of JanusGraph predicates
+
+Support for old serialization format of JanusGraph predicates, client compatibility with 0.6 or higher
```

Comment:
This sentence looks incomplete. It would also be good to name the affected I/O formats Gryo and GraphSON.